### PR TITLE
fix Instance fields accessor

### PR DESCRIPTION
**Pull request by danrubel**
@devoncarew 
+ fix spec so that Instance.getFields() method returns array of BoundFields
+ update VmServiceTest to print PlainInstance objects
+ rename VMServiceTest --> VmServiceTest

### CODE REVIEW

File: dart/tool/service.md [hunk 1]
```diff
@@ -1362,7 +1362,7 @@ class Instance extends Object {
   @Class parameterizedClass [optional];
 
   // The fields of this Instance.
-  BoundField fields [optional];
+  BoundField[] fields [optional];
```
Comment (devoncarew) [inline dart/tool/service.md]: @danrubel, we need to upstream this change. Can you open an issue or a PR?

Comment (devoncarew) [inline dart/tool/service.md]: Also, good catch :)

**Comment by devoncarew** [comment]
lgtm

File: dart/tool/service.md (hunk 1 above)
Comment (danrubel) [inline dart/tool/service.md]: PR https://github.com/dart-lang/sdk/pull/24481

**Pull request merged**